ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1721,3 +1724,19 @@ enum pkcs11_rc set_key_data(struct obj_attrs **head, void *data,
 		return PKCS11_CKR_GENERAL_ERROR;
 	}
 }
+
+enum pkcs11_rc get_key_data_to_wrap(struct pkcs11_object *obj, void *data,
+				    size_t *sz)
+{
+	switch (get_class(obj->attributes)) {
+	case PKCS11_CKO_SECRET_KEY:
+		if (get_attribute_ptr(obj->attributes, PKCS11_CKA_VALUE, data,
```

Comment:
get_attribute() expects a `void **` as 3rd arg.

I think get_key_data_to_wrap() should get a `void **data` argument and we remove the cast when this function is valled.

Reply:
yes, you are right. I will fix this.